skimage/util/dtype.py

```diff
@@ -242,6 +251,13 @@ def _convert(image, dtype, force_copy=False, uniform=False):
         https://github.com/scikit-image/scikit-image/issues/543#issuecomment-208202228
         https://github.com/scikit-image/scikit-image/pull/3575
 
+    legacy_float_range : bool, optional
+        By default and if ``False``, the contents of integer images will be
```

Comment:
```suggestion
        By default the contents of integer images are
```

Reply:
Note, I tweaked your suggestion slightly. It feels weird not to mention the value only indirectly. But not married to it, if you feel strongly about it. :wink: